spanner-ITs-against-emulator.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License..
+
+# Fail on any error.
+set -e
+
+export SPANNER_EMULATOR_HOST=localhost:9010
+export TEST_PROJECT=emulator-test-project
+echo "Running the Cloud Spanner emulator: $SPANNER_EMULATOR_HOST";
+
+# Download the Google Cloud SDK and fetch the beta component which contains the
+# emulator.
+GOOGLE_CLOUD_SDK_VERSION=299.0.0
+wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_CLOUD_SDK_VERSION}-linux-x86_64.tar.gz
+tar zxvf google-cloud-sdk-${GOOGLE_CLOUD_SDK_VERSION}-linux-x86_64.tar.gz
+
+export CLOUDSDK_CORE_DISABLE_PROMPTS=1
+./google-cloud-sdk/bin/gcloud config set disable_usage_reporting false
```

Comment:
Why do we want to enable usage reporting for these tests?

Reply:
Oh, it's because otherwise there was a prompt. But I think it's fine if it defaults to Yes. I'll remove this.